plugins/Live/lang/en.json

```
@@ -1,69 +1,69 @@
 {
     "Live": {
-        "AveragePageGenerationTime": "Each page took on average %1$s to load for this visitor.",
+        "AveragePageGenerationTime": "Each page took %1$s to load for this visitor on average.",
         "CalculatedOverNPageViews": "Calculated using this visitor's last %1$s page views.",
-        "ClickToViewMoreAboutVisit": "Click to view more information about this visit",
-        "ClickToViewAllActions": "Click to view all actions of this group in detail",
+        "ClickToViewMoreAboutVisit": "Click for more info about this visit",
+        "ClickToViewAllActions": "Click for all actions of this group in detail",
         "ConvertedNGoals": "Converted %s Goals",
         "FirstVisit": "First visit",
         "GoalType": "Type",
         "HideMap": "hide map",
-        "KeywordRankedOnSearchResultForThisVisitor": "The keyword %1$s was ranked %2$s on the %3$s search result page for this visitor",
+        "KeywordRankedOnSearchResultForThisVisitor": "The keyword \"%1$s\" was ranked %2$s on the %3$s search result page for this visitor",
         "LastHours": "Last %s hours",
         "LastMinutes": "Last %s minutes",
         "LastVisit": "Last visit",
-        "LinkVisitorLog": "View detailed visits log",
-        "LoadMoreVisits": "Load more visits",
-        "LimitedSummary": "The information in this profile summarizes the last %1$s visits. This user had more visits in total.",
+        "LinkVisitorLog": "View detailed visit log",
+        "LoadMoreVisits": "Load more",
+        "LimitedSummary": "The info in this profile summarizes the last %1$s visits. This user had more visits in total.",
         "LimitedVisitsShown": "Only the last %1$s visits are shown. This user had more visits in total.",
         "MorePagesNotDisplayed": "more pages by this visitor are not displayed",
         "NbVisitor": "1 visitor",
         "NbVisitors": "%s visitors",
         "NextVisitor": "Next visitor",
-        "NoMoreVisits": "There are no more visits for this visitor.",
+        "NoMoreVisits": "These are all the visits for this visitor.",
         "PageRefreshed": "Number of times this page was viewed \/ refreshed in a row.",
-        "PluginDescription": "Provides the live Visitor Log and lets you watch your visitors live in the real-time dashboard widget. The plugin also lets you see a Visitor profile for any given user.",
+        "PluginDescription": "Provides the live visitor log and lets you watch your visitors live in the real-time dashboard widget. A visitor profile can be browsed for any given user.",
         "PreviousVisitor": "Previous visitor",
         "RealTimeVisitors": "Real-time Visits",
-        "RealTimeVisitorCount": "Real Time Visitor Count",
+        "RealTimeVisitorCount": "Real-time Visitor Count",
```

Shouldn't that be:
```suggestion
        "RealTimeVisitorCount": "Real-time visitor count",
```
Or is there a reason for having the capital letters?